client/windows/CMapOverview.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * CMapOverview.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+VCMI_LIB_NAMESPACE_BEGIN
+class CMap;
+VCMI_LIB_NAMESPACE_END
+#include "CWindowObject.h"
+#include "../../lib/filesystem/ResourcePath.h"
+#include "../gui/InterfaceObjectConfigurable.h"
+
+class CSlider;
+class CLabel;
+class CPicture;
+class CFilledTexture;
+class CTextBox;
+class IImage;
+class Canvas;
+class TransparentFilledRectangle;
+enum ESelectionScreen : ui8;
+
+class CMapOverview : public CWindowObject
+{
+	class CMapOverviewWidget : public InterfaceObjectConfigurable
```

Comment:
Personally prefer to avoid embedded classes, if only to avoid more complex declaration form and awkward `CMapOverview::CMapOverviewWidget::CMapOverviewWidget` syntax for methods, but not critical